go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/beorn7/perks v1.0.1
 	github.com/cespare/xxhash/v2 v2.1.2
 	github.com/golang/protobuf v1.5.2
+	github.com/jpillora/backoff v1.0.0 // indirect
```

Comment:
This looks off. Do you mind doing `go mod tidy`?

Reply:
yeah that got added with `go mod tidy`

```
go mod why github.com/jpillora/backoff
# github.com/jpillora/backoff
github.com/prometheus/client_golang/api/prometheus/v1
github.com/prometheus/client_golang/api/prometheus/v1.test
github.com/prometheus/common/config
github.com/mwitkow/go-conntrack
github.com/jpillora/backoff
```